spark/src/test/scala/org/apache/spark/sql/delta/managedcommit/CommitOwnerClientSuite.scala

```diff
@@ -218,4 +220,54 @@ class CommitOwnerClientSuite extends QueryTest with DeltaSQLTestUtils with Share
     assert(obj1 != obj3)
     assert(!obj1.semanticEquals(obj3))
   }
+
+  private def checkMissing[Interface: TypeTag, Class: TypeTag](): Set[String] = {
+    val fields = typeOf[Class].decls.collect {
+      case m: MethodSymbol if m.isCaseAccessor => m.name.toString
+    }
+
+    val getters = typeOf[Interface].decls.collect {
+      case m: MethodSymbol if m.isAbstract => m.name.toString
+    }.toSet
+
+    fields.filterNot { field =>
+      getters.contains(s"get${field.capitalize}")
+    }.toSet
+  }
+
+  /**
+   * We expect the Protocol action to have the same fields as AbstractProtocol (part of the
+   * CommitStore interface). With this if any change has happened in the Protocol of the table,
```

Comment:
CommitStore has been replaced with Commit owner / ComitOwnerClient -- depending on the context in the entire code/comments. Could we follow the same convention here as well.

Reply:
Updated to reflect the new naming.